tests/Makefile.am

```diff
@@ -65,6 +65,7 @@ EXTRA_DIST += data/SPECS/symlinktest.spec
 EXTRA_DIST += data/SPECS/deptest.spec
 EXTRA_DIST += data/SPECS/verifyscript.spec
 EXTRA_DIST += data/SPECS/fakeshell.spec
+EXTRA_DIST += data/SPECS/mini.spec
```

Comment:
didn't you forgot to commit the actual file?

Reply:
So it seems...